src/components/ExternalLink.tsx

```diff
@@ -26,13 +26,13 @@
 
 import React from "react"
 import { Link, LinkProps } from "@looker/components"
-import { ExtensionContext } from "@looker/extension-sdk-react"
+import { ExtensionContext2 } from "@looker/extension-sdk-react"
 
 export const ExternalLink: React.FC<Omit<LinkProps, "color">> = (
   props: any
 ) => {
   return (
-    <ExtensionContext.Consumer>
+    <ExtensionContext2.Consumer>
```

Comment:
Ugh!
Try this as an alternative (I have not tested but if you are in here let's clean this up
```
import React, {useContext} from "react"
import { Link, LinkProps } from "@looker/components"
import { Looker40SDK } from '@looker/sdk'
import {
  ExtensionContext2,
  ExtensionContextData2,
} from '@looker/extension-sdk-react'

export const ExternalLink: React.FC<Omit<LinkProps, "color">> = (
  {href, target, onClick, ...rest}
) => {
  const extensionContext = useContext<ExtensionContextData2<Looker40SDK>>(
    ExtensionContext2
  )
  const { extensionSDK } = extensionContext
  const onLinkClick = (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
    // probably don't need to call this as href is no longer added to the Link
    event.stopPropagation()
    if (href) {
      extensionSDK.updateLocation(
        href,
        undefined,
        target
      )
    } else if (onClick) {
      onClick(event)
    }
  }

  return (
    <Link onClick={onLinkClick} {...rest} />
  )
}
```

Reply:
Clutch code snippet! This works for me. I stole the previous one from DD